HiP-DataStore/Core/WriteModel/Commands/ExhibitPageCommands.cs

```diff
@@ -70,67 +70,29 @@ public static class ExhibitPageCommands
             }
         }
 
-        private static IEnumerable<IEvent> AddExhibitPageReferences(int pageId, ExhibitPageArgs2 args)
-        {
-            if (args.Audio != null)
-                yield return new ReferenceAdded(ResourceType.ExhibitPage, pageId, ResourceType.Media, args.Audio.Value);
-
-            if (args.Image != null)
-                yield return new ReferenceAdded(ResourceType.ExhibitPage, pageId, ResourceType.Media, args.Image.Value);
-
-            foreach (var img in args.Images?.Distinct() ?? Enumerable.Empty<SliderPageImageArgs>())
-                yield return new ReferenceAdded(ResourceType.ExhibitPage, pageId, ResourceType.Media, img.Image);
-
-            foreach (var id in args.AdditionalInformationPages?.Distinct() ?? Enumerable.Empty<int>())
-                yield return new ReferenceAdded(ResourceType.ExhibitPage, pageId, ResourceType.ExhibitPage, id);
-        }
-
-        private static IEnumerable<IEvent> RemoveExhibitPageReferences(int pageId, ReferencesIndex referencesIndex)
-        {
-            return referencesIndex
-                .ReferencesOf(ResourceType.ExhibitPage, pageId)
-                .Select(reference => new ReferenceRemoved(ResourceType.ExhibitPage, pageId, reference.Type, reference.Id));
-        }
-
         public static IEnumerable<IEvent> Create(int pageId, ExhibitPageArgs2 args)
         {
-            var ev = new ExhibitPageCreated3
+            yield return new ExhibitPageCreated3
```

Comment:
That's because until now the method returned multiple events. But you are right, I should change the return type then. I could even get rid of the method completely, as it is only used once.